Buildings/Experimental/DHC/CentralPlants/Cooling/Controls/Validation/ChillerStage.mo

```diff
@@ -0,0 +1,65 @@
+within Buildings.Experimental.DHC.CentralPlants.Cooling.Controls.Validation;
+model ChillerStage
+  "Example to test the chiller staging controller"
+  extends Modelica.Icons.Example;
+  Buildings.Experimental.DHC.CentralPlants.Cooling.Controls.ChillerStage chiStaCon(
+    tWai=30,
+    QChi_nominal=-200*3.517*1000)
+    "Chiller staging controller"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}})));
+  Modelica.Blocks.Sources.BooleanTable on(
+    table(
+      each displayUnit="s")={300,900})
+    "On signal of the cooling plant"
+    annotation (Placement(transformation(extent={{-60,40},{-40,60}})));
+  Modelica.Blocks.Sources.Sine mFlo(
+    amplitude=0.5*chiStaCon.QChi_nominal/(-10)/4200,
+    freqHz=1/300,
+    offset=0.5*chiStaCon.QChi_nominal/(-10)/4200)
+    "Total mass flow rate"
+    annotation (Placement(transformation(extent={{-60,-50},{-40,-30}})));
+  Modelica.Blocks.Sources.Constant TRet(
+    k=18)
```

Comment:
I know the Modelica FMT broke this onto multiple lines, but I wonder if we should keep it on one line since it is short? The MBL hasn't fully adopted the FMT yet. There are some other instances like this as well.

Reply:
Sure, I changed TRet along with TSup.